docs/DeveloperGuide.adoc

```diff
@@ -1,4 +1,4 @@
-= AddressBook Level 4 - Developer Guide
+= Club Connect - Developer Guide
```

Comment:
Good to see that you all have updated your DevGuide extensively, this will save you quite a bit of time toward the end. 👏